docs/apis/readEndpoint.md

```diff
@@ -0,0 +1,45 @@
+## createEntity(value: { object })
```

Comment:
That must have slipped when i've had copy pasted various structure over. Nice catch! :)